packages/components/table/src/table-header/index.ts

```diff
@@ -57,6 +57,10 @@ export default defineComponent({
         }
       },
     },
+    appendFilterPanelTo: {
+      type: String,
+      default: 'body',
```

Comment:
The default value should probably be removed here too.

Reply:
done